### DISCUSSION TITLE
What do you think about support Language Server Protocol?

### DISCUSSION THREAD

**Discussion by nsgnkhibdk2cls0f** [stefankueng/BowPad#303] [General]
Supporting LSP will transform your text editor to an IDE of many programming languages (think of something like VSCode). SublimeText now also supports LSP. Various other free and open source text editors also support LSP:

https://github.com/orbitalquark/textadept-lsp

https://github.com/lite-xl/lite-xl-lsp

**Comment by stefankueng** [comment]
created an issue for this: https://github.com/stefankueng/BowPad/issues/304

**Reply by goyalyashpal** [reply]
hi @ stefankueng 
u don't have to use "reference in new issue" button to create issue from discussions. 

github has this feature built-in, titled: "create issue from discussion" (and also, vice-versa: "convert issue into a discussion"
the output of "create issue from discussion" is slightly different & better than "reference in new issue"

![image](https://github.com/stefankueng/BowPad/assets/19423063/dbf890d7-025a-4128-a220-9f81aedec680)



![image](https://github.com/stefankueng/BowPad/assets/19423063/16435eb2-ade3-4b22-8579-eec8bbf3f566)


![image](https://github.com/stefankueng/BowPad/assets/19423063/dc43e4d2-1817-42ac-8032-4d099a38217c)
